packaging/Gemfile

```diff
@@ -1,13 +1,13 @@
 source 'http://rubygems.org'
 
-gem "pact", "1.62.0"
+gem "pact", "1.63.0"
 gem "pact-message", "0.11.1"
-gem "pact-mock_service", "3.9.0"
+gem "pact-mock_service", "3.11.0"
 gem "pact-provider-verifier", "1.36.1"
-gem "pact_broker-client", "1.65.0"
-gem "webrick", "1.6.1" # Later versions require Ruby 2.3
-gem "rack", "~>2.2.6"
-gem "json", "2.0.4"
-
+gem "pact_broker-client", "1.66.1"
+gem "webrick", "1.8.1"
+gem 'rack', '>= 2.2.6'
+gem "json", "2.6.3"
+gem 'rack-test', '~> 2.1'
```

Comment:
This line can just be removed now, and it can get its version requirement from the gem.